bin/faucet/Cargo.toml

```diff
@@ -23,12 +22,15 @@ actix-web = "4"
 async-mutex = "1.4.0"
 derive_more = "0.99.17"
 figment = { version = "0.10", features = ["toml", "env"] }
-miden-client = { version = "0.3", features = ["concurrent"] }
+miden-client = { version = "0.3" }
 miden-lib = { version = "0.3" }
 miden-node-proto = { workspace = true }
 miden-node-utils = { workspace = true }
 miden-objects = { version = "0.3" }
+miden-tx = { version = "0.3" }
```

Comment:
Should we not import this with `concurrent` feature enabled? (to make transaction proving faster)

Reply:
Done.